multi-cluster-orchestrator/samples/hello_world/README.md

```diff
@@ -0,0 +1,420 @@
+# Hello World Sample
+
+In this quickstart, you learn how to deploy a Hello World sample as a
+multi-region [Google Kubernetes Engine (GKE)][0] workload using
+[Multi-cluster Orchestrator][18] and [Multi-cluster Gateway][19].
+
+Infrastructure as Code (IaC) is a practice of managing and provisioning software
+infrastructure resources using code. Terraform is a popular open source IaC tool
+that supports a wide range of Cloud services, including GKE. As a GKE platform
+administrator, you can use Terraform to standardize configuration of your
+Kubernetes clusters and streamline your DevOps workflows. To learn more, see
+[Terraform support for GKE][1].
+
+## Objectives
+
+- Deploy the pre-requisite infrastructure, including:
+  - Hub GKE cluster running Multi-cluster Orchestrator and Argo CD
+  - Worker GKE clusters in multiple-regions
+- Deploy the Hello World sample workload using Multi-cluster Orchestrator and
+  Argo CD
+- Load test the Hello World server using Multi-cluster Gateway to trigger multi-cluster scaling
+
+## Before you begin
+
+Take the following steps to enable the Kubernetes Engine API:
+
+  1. In the Google Cloud console, on the project selector page, select or [create a
+  Google Cloud project][2].
+
+  > [!TIP]
+  > If you don't plan to keep the resources that you create in this procedure,
+  > create a project instead of selecting an existing project. After you finish
+  > these steps, you can delete the project, removing all resources associated with
+  > the project.
+
+1. [Make sure that billing is enabled for your Google Cloud
+  project][3].
+
+1. [Enable the following services][4]:
```

Comment:
is that a pre-req before running TF? I'm surprised you don't do it in TF (or would it make the ordering annoying?)

Reply:
I dropped for docs style, but as this is for GH I can re-add.  The ordering is OK but requires several `depends_on` :(